### PR TITLE
Improve the environment information for bug report

### LINKED ISSUE #1096: Improve the environment information for bug report
**What would you like to be added/modified**:

**Why is this needed**:

now our bug report environment is:
```
**Environment**:
- KubeEdge version:
- Hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Others:


```

I recommend dividing the environment information into cloud and side

```
**CloudSide Environment**:
- KubeEdge version(e.g. \`keadm version -o json\` or `cloudcore --version`):   
- Hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Others:

**EdgeSide Environment**:
- KubeEdge version(e.g. \`keadm version -o json\` or `edgecore --version`): 
- Hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Others:

```

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1096 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -7,6 +7,25 @@ labels: kind/bug
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 
+**Environment**:
+- KubeEdge version(e.g. `keadm version -o json`):
```
Comment (fisherxu) [inline .github/ISSUE_TEMPLATE/bug-report.md]:
```suggestion
- KubeEdge version(e.g. `cloudcore/edgecore --version`):  
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -7,6 +7,25 @@ labels: kind/bug
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 
+**Environment**:
+- KubeEdge version(e.g. `keadm version -o json`):  
+
+**CloudSide Environment**:
+- cloudcore version (e.g. `cloudcore --version`):
```
Comment (fisherxu) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Remove this line? We need a unify version of kubeedge in cloud and edge side.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -7,6 +7,25 @@ labels: kind/bug
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 
+**Environment**:
```
Comment (fisherxu) [inline .github/ISSUE_TEMPLATE/bug-report.md]: How about keep the `Environment` at the bottom

Comment (kadisi) [reply .github/ISSUE_TEMPLATE/bug-report.md]: ok

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (kadisi) [reply .github/ISSUE_TEMPLATE/bug-report.md]: done

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (kadisi) [reply .github/ISSUE_TEMPLATE/bug-report.md]: done

**Pull request merged**